package.json

```diff
@@ -93,7 +93,7 @@
     "webpack-merge": "^5.8.0"
   },
   "dependencies": {
-    "@centreon/centreon-frontend": "git+https://centreon@github.com/centreon/centreon-frontend.git#develop",
+    "centreon-frontend": "git+https://centreon@github.com/centreon/centreon-frontend.git#Fix-path-for-Jest",
```

Comment:
Don't forget to re-use the `develop` branch

Reply:
^^ yes